eng/tools/rush-runner/src/rush.js

```diff
@@ -52,6 +53,32 @@ export function rushRunAllWithDirection(action, packagesWithDirection, rushParam
     invocation,
   });
 
+  // Restore assets for packages that are being 'unit-test'-ed in the CI pipeline
+  if (
+    // 1. Check if running in Public CI (process.env["BUILD_BUILDNUMBER"] is set)
+    process.env["BUILD_BUILDNUMBER"]
+    // 2. Ensure not in "live" or "record" mode (run only in playback mode)
+    && (!["live", "record"].includes(process.env.TEST_MODE))
+    // 3. Ensure the action is either 'unit-test:node' or 'unit-test:browser' (unit tests)
+    && (['unit-test:node', 'unit-test:browser'].includes(action))
+  ) {
+    console.log(`Running rush list with ${invocation.join(" ")}`);
+
+    // Get the list of packages to run the action on
+    const output = spawnNodeWithOutput(
```

Comment:
should we put `spawnNodeWithOutput` inside a try-catch and log then ignore the error?

Reply:
good idea